Known issue for UI logins with underscores and unauth listing (#31140)
hashicorp/vault: website/content/docs/updates/important-changes.mdx

## Known issues

### UI login fails for auth mounts with underscores and unauthenticated listing ((#ui-login-underscore))

| Change       | Affected version | Fixed version
| ------------ | ---------------- | --------------------

> | Known issue  | 1.20.0           | N/A

Author: I wasn't sure about this either, but I'm sure we'll forget to update this when 1.20.1 gets released, so we might as well add the fixed version now.

> | Known issue  | 1.20.0           | N/A

Reviewer: Great point 😅

If an auth method is mounted at a path that includes an underscore (e.g. oidc_test)
**and** the mount is tuned with `listing_visibility="unauth"` the login request will
fail because the UI requests the wrong endpoint. Mounts with dashes, e.g. oidc-test
still log in fine.

#### Recommendation

As a workaround, you can log in to the UI using the following steps:
1. Navigate directly to the URL for that method’s type `${VAULT_ADDR}/ui/vault/auth?with=oidc`

> 2. Click "Sign in with other methods ->"

Reviewer 2: ```suggestion
1. Click "Sign in with other methods ->"
```
Markdown correction: use `1.` for all numbered list items

> 3. Select type from the dropdown

Reviewer 2: ```suggestion
1. Select type from the dropdown
```
Markdown correction: use `1.` for all numbered list items

Question: is "type" a GUI label or do we mean they should select the appropriate type from the dropdown?

> 4. Click "Advanced settings" and input the correct path, e.g. oidc_test

Reviewer 2: ```suggestion
1. Click **Advanced settings** and provide the correct path (e.g., `oidc_test`).
```
Markdown correction: use `1.` for all numbered list items
